src/app/enrolled-course-details-page/enrolled-course-details-page.html

```diff
@@ -138,7 +138,7 @@
 			<ion-card *ngSwitchCase="'info'">
 				<ion-card-content class="ion-no-padding">
 					<div class="ion-padding">
-					<div class="certificate-container">
+					<div *ngIf="course?.progress && course?.progress === 100" class="certificate-container">
```

Comment:
This condition should not be on the basis of `progress` it should be on the basis of  `certificate`